src/sentry/static/sentry/app/components/events/interfaces/threads/threadSelector/selectedOption.tsx

```diff
@@ -3,38 +3,38 @@ import styled from '@emotion/styled';
 
 import TextOverflow from 'app/components/textOverflow';
 import space from 'app/styles/space';
+import {tct, t} from 'app/locale';
 
 type Props = {
   id: string;
   details: ThreadInfo;
 };
 
 type ThreadInfo = {
-  label: string;
+  label?: string;
   filename?: string;
 };
 
-const ThreadsSelectorSelectedOption = ({id, details}: Props) => (
+const SelectedOption = ({id, details}: Props) => (
   <Wrapper>
-    <StyledThreadID>{`Thread #${id}:`}</StyledThreadID>
-    <StyledOptionLabel>{details.label}</StyledOptionLabel>
+    <ThreadId>{tct('Thread #[id]:', {id})}</ThreadId>
+    <Label>{details?.label || t('<unknown>')}</Label>
```

Comment:
Hmm, so generally we include punctuation in translations which I think is fine -- but in this case it's less punctuation and more of a visual indicator that this is a special value. In this case I would only translate `unknown`, or not even translate it at all.

```suggestion
    <Label>{details?.label || `<${t('unknown')}>`}</Label>
```